Source/Extensions/UIBezierPathExtension.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  UIBezierPathExntension.swift
+//  
+//
+//  Created by Ehab Alsharkawy on 14/11/2022.
+//
+
+import UIKit
+
+extension UIBezierPath {
+
+    func drawLine(startX: CGFloat, startY: CGFloat, endX: CGFloat, endY: CGFloat) {
+        let startPoint = CGPoint(x: startX, y: startY)
+        let endPoint = CGPoint(x: endX, y: endY)
+        move(to: startPoint)
+        addLine(to: endPoint)
+    }
+
+    func drawCorner(startX: CGFloat, startY: CGFloat, endX: CGFloat, endY: CGFloat, controlX: CGFloat, controlY: CGFloat) {
```

Comment:
I think we should be able to test these methods? I haven't tried this code, just checking the APIs, so might need changes to make work. But appears possible?

```swift
let testStartX: CGFloat = 0
let testStartY: CGFloat = 12
let testEndX: CGFloat = 78
let testEndY: CGFloat = 576

// Use method to create path
let testPath = UIBezierPath()
testPath.drawLine(startX: testStartX, .... )
let testCGPath = testPath.cgPath

// Prepare standard behaviour
let expectedPath = UIBezierPath()
expectedPath.move(to: CGPoint(x: testStartX, y: testStartY))
expectedPath.addLine(to: CGPoint(x: testEndX, y: testEndY))
let expectedCGPath = expectedPath.cgPath

// Ensure tested method has same outcome
let diffPath = testCGPath.symmetricDifference(expectedCGPath)
XCTAssertTrue(diffPath.isEmpty)
```